packages/venia-concept/src/components/App/app.js

```diff
@@ -12,9 +12,9 @@ import Icon from 'src/components/Icon';
 
 import { getToastId, useToasts } from '@magento/peregrine';
 
-import AlertCircleIcon from 'react-feather/dist/icons/alert-circle';
-import CloudOffIcon from 'react-feather/dist/icons/cloud-off';
-import WifiIcon from 'react-feather/dist/icons/wifi';
+import { AlertCircle as AlertCircleIcon } from 'react-feather';
+import { CloudOff as CloudOffIcon } from 'react-feather';
+import { Wifi as WifiIcon } from 'react-feather';
```

Comment:
I could now combine these into a single import with csv -- just kept them individual for diff sake.

Reply:
Combining would be nice. 👍

Reply:
I can do that -- the truth is I used a regex replace for these to make the initial conversion :D